watertap/unit_models/electroNP_ZO.py

```diff
@@ -13,91 +13,43 @@
 # Import Pyomo libraries
 from pyomo.environ import (
     Var,
-    check_optimal_termination,
     Param,
     Suffix,
     NonNegativeReals,
     units as pyunits,
 )
-from pyomo.common.config import ConfigBlock, ConfigValue, In
+from idaes.models.unit_models.separator import SeparatorData, SplittingType
 
 # Import IDAES cores
 from idaes.core import (
     declare_process_block_class,
-    UnitModelBlockData,
-    useDefault,
 )
-from idaes.core.solvers import get_solver
+
 from idaes.core.util.tables import create_stream_table_dataframe
-from idaes.core.util.config import is_physical_parameter_block
-from idaes.core.util.exceptions import ConfigurationError, InitializationError
+from idaes.core.util.exceptions import ConfigurationError
 import idaes.core.util.scaling as iscale
 import idaes.logger as idaeslog
 
-from watertap.core import InitializationMixin
 
 __author__ = "Chenyu Wang"
 
 _log = idaeslog.getLogger(__name__)
 
 
 @declare_process_block_class("ElectroNPZO")
-class ElectroNPZOData(InitializationMixin, UnitModelBlockData):
+class ElectroNPZOdata(SeparatorData):
     """
     Zero order electrochemical nutrient removal (ElectroNP) model based on specified water flux and ion rejection.
```

Comment:
I know this doc string is old, but "specified water flux and ion rejection" seems inaccurate. Should be something like "based on specified removal efficiencies for nitrogen and phosphorus."